source/walkthroughs/deploy/install_language_runtime/_nodejs.md.erb

```diff
@@ -3,7 +3,7 @@
 
 <% if language_type == :meteor %>
 <div class="note">
-  Meteor 1.4 and newer require Node.js 4 (LTS), however previous versions of Meteor &lt; 1.4 <a href="https://github.com/meteor/meteor/issues/5124">require an older Node.js version (0.10)</a>. This is reflected in our installation instructions below.<br>
+  Meteor 1.6 and newer require Node.js 8, however previous versions of Meteor require an older Node.js version, Node.js 0.10 for <a href="https://github.com/meteor/meteor/pull/7218">Meteor &lt; 1.4</a> and Node.js 4.4 for <a href="https://github.com/meteor/meteor/pull/8728">1.4 &lt;= Meteor &lt; 1.6</a>. This is reflected in our installation instructions below.<br>
```

Comment:
This sentence got a bit complex. I would suggest replacing this and the picker start with something like:

Different versions of Meteor require different versions of Node.js. Please select your Meteor version to get the correct installation instructions for Node.js.